Tests/MapboxNavigationTests/NavigationCameraTests.swift

```diff
@@ -461,7 +461,11 @@ class NavigationCameraTests: TestCase {
         let expectedCenterCoordinate = CLLocationCoordinate2D(latitude: 0.0, longitude: 0.0)
         navigationViewportDataSource.followingMobileCamera.center = expectedCenterCoordinate
         
-        let expectedZoom: CGFloat = 11.1
+        
+        // Since zoomUpdatesAllowed is disabled it is expected that `CameraOptions.zoom`, which was
+        // returned from the `ViewportDataSourceDelegateMock` will be the median value of the zoom range.
+        let zoomRange = navigationViewportDataSource.options.followingCameraOptions.zoomRange
```

Comment:
Please also make sure to update `OverviewCameraOptions` to also have default values.